KITE-Extras/src/main/java/io/cosmosoftware/kite/steps/TestStep.java

```diff
@@ -56,5 +59,23 @@ public AllureStepReport getStepReport() {
   protected abstract void step() throws KiteTestException;
 
   public void setLogger(Logger logger) { this.logger = logger; }
-
+  
+  protected String getClassName() {
+    
+    String name = this.getClass().getSimpleName();
+    Set<String> upperLetters = new HashSet<>();
+    
+    for (char letter: name.toCharArray()) {
+      String letterString = Character.toString(letter);
+      if (letterString.matches("[A-Z]") || letterString.matches("[0-9]")) {
+        upperLetters.add(letterString);
+      }
+    }
+    
+    for (String letterString: upperLetters) {
+      name = name.replaceAll(letterString, " " + letterString.toLowerCase());
+    }
+    
+    return name;
```

Comment:
why so complicated?

Reply:
I wanted to change 'AClassNameThatLookLikeThis' to 'A Class Name That Look Like This' for logging.